src/engraving/dom/stafftypechange.cpp

```diff
@@ -133,9 +133,21 @@ bool StaffTypeChange::setProperty(Pid propertyId, const PropertyValue& v)
     case Pid::STEP_OFFSET:
         m_staffType->setStepOffset(v.toInt());
         break;
-    case Pid::STAFF_LINES:
-        m_staffType->setLines(v.toInt());
-        break;
+    case Pid::STAFF_LINES: {
```

Comment:
Small codestyle thing: we usually put the brace in the new line in these cases

Reply:
Ok. So I changed also other similar places.